README.md

```diff
@@ -28,36 +28,9 @@ Git on Windows will, by default, automatically convert line endings from `lf` to
 
 See [this writeup](https://help.github.com/articles/dealing-with-line-endings/) for more information.
 
-2. Create the TFB Docker virtual network
+2. Run a test.
 
-        $ docker network create tfb
-
-3. Run a test.
-
-        $ docker run -it --network=tfb -v /var/run/docker.sock:/var/run/docker.sock --mount type=bind,source=[ABS PATH TO THIS DIR],target=/FrameworkBenchmarks techempower/tfb --mode verify --test gemini
-
-### Explanation of the run script
-
-That run script is pretty wordy, but each and every flag is required. Unfortunately, because of the way that Docker runs processes, you **cannot** put this inside of a shell script without breaking how `ctrl+c` and `SIGTERM` work (the shell script would receive the signal, do nothing with the underlying python suite running, and exit, orphaning the toolset to continue running).
-
-- `-it` tells docker to run this in 'interactive' mode and simulate a TTY, so that `ctrl+c` is propagated.
-- `--network=tfb` tells the container to join the 'tfb' Docker virtual network
-- `-v` specifies which Docker socket path to mount as a volume in the running container. This allows docker commands run inside this container to use the host container's docker to create/run/stop/remove containers.
-- `--mount` mounts the FrameworkBenchmarks source directory as a volume to share with the container so that rebuilding the toolset image is unnecessary and any changes you make on the host system are available in the running toolset container.
-- `techempower/tfb` is the name of toolset container to run
-- `--mode verify --test gemini` are the command to pass to the toolset.
-
-#### A note on Linux:
-
-You may not want to call step 4 from above every time. You can add an `alias` to your `~/.bash_aliases` file to shorten it since it will not change once configured:
-
-`$ alias tfb="docker run -it --network=tfb -v /var/run/docker.sock:/var/run/docker.sock --mount type=bind,source=[ABS PATH TO THIS DIR],target=/FrameworkBenchmarks techempower/tfb"`
-
-`$ source ~/.bash_aliases`
-
-Now you can run the toolset via `tfb`:
-
-`$ tfb --mode verify --test gemini`
+        $ ./tfb --mode verify --test gemini
```

Comment:
I am fine with this change, but I would like to keep the explanation of the `docker run` command.

Reply:
OK, shall we paste the explanation of the docker run command into the `tfb` script?